fastlane/Fastfile

```diff
@@ -876,6 +876,16 @@ platform :ios do
     update_snapshots_commit(new_hash)
   end
 
+  lane :deploy_to_spm do
+    source_repo = ENV["CIRCLE_REPOSITORY_URL"]
```

Comment:
It's fine to use an environment variable for this, but we could also build the URL here. We have the `repo_name` in this file already so we could just build the URL like `"https://github.com/revenuecat/#{repo_name}"` I think, this way we avoid needing extra environment variables. We could also use the environment variable if it exists but fallback to that, as a way to allow overriding. But as I said, not a blocker.